### PR TITLE
Added TKR query in project

### PR DESCRIPTION
**Pull request by ghulamhafeez**
## TKR query

- Created TKR query example in project

### CODE REVIEW

File: src/services/TKRservices.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+export const TodosApi = createApi({
+    reducerPath: "TodosApi",
+    baseQuery: fetchBaseQuery({
+      baseUrl: "https://jsonplaceholder.typicode.com",
+    }),
+  
+    endpoints: (builder) => ({
+      getTodos: builder.query({
```
Comment (Monfernape) [inline src/services/TKRservices.js:9]: Great.

File: src/views/RTK-Query/RtkData.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { useGetTodosQuery } from "../../services/TKRservices";
+const RtkData = () => {
+  const { data: todosData = [], isLoading } = useGetTodosQuery();
```
Comment (Monfernape) [inline src/views/RTK-Query/RtkData.js:4]: Cool

**Review by Monfernape — APPROVED** [review]
It's RTK query not TRK. Can we rename the PR please?

File: src/services/TKRservices.js (hunk 1 above)
Comment (ghulamhafeez) [reply src/services/TKRservices.js:9]: Thanks

File: src/views/RTK-Query/RtkData.js (hunk 2 above)
Comment (ghulamhafeez) [reply src/views/RTK-Query/RtkData.js:4]: Thanks

**Pull request merged**